README.md

```diff
@@ -8,67 +8,13 @@
 
 ---
 
-
-```mermaid
-  graph LR;
-      inventory[ Account inventory] 
-      api[Impacts from BoaviztaAPI]
-      metrics[Prometheus metrics]
-      json[JSON output]
-      inventory --> api
-      api --> metrics
-      api --> json 
-```
-
-Cloud scanner combines real time _inventory_ and _usage_ data from your AWS account with [Boavizta API](https://github.com/Boavizta/boaviztapi/) to offer a  view of your impacts on a given region.
-
-- multi criteria: Primary Energy consumption (PE), Abiotic resource depletion potential (ADP), and Global Warming Potential (GWP)
-- multi stage: separate impacts of resources Usage and Manufacture
-
-This is similar to what you can get from [Datavizta](http://datavizta.boavizta.org/cloudimpact) but with automated inventory.
```

Comment:
**I would like to keep theses lines:**

Cloud scanner combines real time _inventory_ and _usage_ data from your AWS account with [Boavizta API](https://github.com/Boavizta/boaviztapi/) to offer a  view of your impacts on a given region.

- multi criteria: Primary Energy consumption (PE), Abiotic resource depletion potential (ADP), and Global Warming Potential (GWP)
- multi stage: separate impacts of resources Usage and Manufacture

This is similar to what you can get from [Datavizta](http://datavizta.boavizta.org/cloudimpact) but with automated inventory.